<?xml version="1.0" encoding="utf-8"?>
<calculation-relax-static><key>08df1b35-d3f5-4082-94bc-3d9e93ffc6a1</key><calculation><iprPy-version>0.8.2</iprPy-version><atomman-version>1.2.3</atomman-version><LAMMPS-version>10 Mar 2016-ICMS</LAMMPS-version><script>calc_relax_static</script><run-parameter><size-multipliers><a>0</a><a>10</a><b>0</b><b>10</b><c>0</c><c>10</c></size-multipliers><energytolerance>0.0</energytolerance><forcetolerance><value>1e-10</value><unit>eV/angstrom</unit></forcetolerance><maxiterations>10000</maxiterations><maxevaluations>100000</maxevaluations><maxatommotion><value>0.01</value><unit>angstrom</unit></maxatommotion></run-parameter></calculation><potential-LAMMPS><key>c83296e8-ef62-439e-bd8b-f575625f80a3</key><id>2013--Smirnova-D-E--U-Mo-Xe--LAMMPS--ipr1</id><potential><key>fc5a9a34-b8be-4935-aa63-aa02a15d93b8</key><id>2013--Smirnova-D-E-Kuksin-A-Y-Starikov-S-V-et-al--U-Mo-Xe</id></potential></potential-LAMMPS><system-info><family>A1--Cu--fcc</family><artifact><file>30cfbeff-9333-4fbd-b05e-1c98198d233c.json</file><format>system_model</format><load_options>key minimum-atomic-system index 4</load_options></artifact><symbol>U</symbol></system-info><phase-state><temperature><value>0.0</value><unit>K</unit></temperature><pressure-xx><value>0.0</value><unit>GPa</unit></pressure-xx><pressure-yy><value>0.0</value><unit>GPa</unit></pressure-yy><pressure-zz><value>0.0</value><unit>GPa</unit></pressure-zz><pressure-xy><value>0.0</value><unit>GPa</unit></pressure-xy><pressure-xz><value>0.0</value><unit>GPa</unit></pressure-xz><pressure-yz><value>0.0</value><unit>GPa</unit></pressure-yz></phase-state><status>error</status><error>b'Traceback (most recent call last):\r\n  File "C:\\Users\\lmh1\\AppData\\Local\\Continuum\\anaconda3\\lib\\shutil.py", line 544, in move\r\n    os.rename(src, real_dst)\r\nFileNotFoundError: [WinError 2] The system cannot find the file specified: \'10000.dump\' -&gt; \'relax_static-0.dump\'\r\n\r\nDuring handling of the above exception, another exception occurred:\r\n\r\nTraceback (most recent call last):\r\n  File "calc_relax_static.py", line 380, in &lt;module&gt;\r\n    main(*sys.argv[1:])\r\n  File "calc_relax_static.py", line 62, in main\r\n    ctol = input_dict[\'cycletolerance\'])\r\n  File "calc_relax_static.py", line 231, in relax_static\r\n    shutil.move(last_dump_file, renamed_dump_file)\r\n  File "C:\\Users\\lmh1\\AppData\\Local\\Continuum\\anaconda3\\lib\\shutil.py", line 558, in move\r\n    copy_function(src, real_dst)\r\n  File "C:\\Users\\lmh1\\AppData\\Local\\Continuum\\anaconda3\\lib\\shutil.py", line 257, in copy2\r\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\r\n  File "C:\\Users\\lmh1\\AppData\\Local\\Continuum\\anaconda3\\lib\\shutil.py", line 120, in copyfile\r\n    with open(src, \'rb\') as fsrc:\r\nFileNotFoundError: [Errno 2] No such file or directory: \'10000.dump\'\r\n'</error></calculation-relax-static>